assets/impersonation-migrations/2020_05_15_000010_create_tenant_user_impersonation_tokens_table.php

```diff
@@ -23,7 +23,7 @@ public function up(): void
             $table->string('redirect_url');
             $table->timestamp('created_at');
 
-            $table->foreign('tenant_id')->references('id')->on('tenants')->onUpdate('cascade')->onDelete('cascade');
+            $table->foreign('tenant_id')->references('id')->on('tenants')->onUpdate('cascade');
```

Comment:
Probably revert this change. We only care about logic for domains, not really about impersonation tokens. Those can disappear with no consequences.